### PR TITLE
Improvements to Compass Rose and HUD

### PR DESCRIPTION
**Pull request by davetsay**
* Flatten HUD (not intended for horizon line)
* Lock bezel should lock to camera FOV, not rover heading
* Lock bezel should persist for imagery view lifetimed

### CODE REVIEW

File: src/plugins/imagery/components/Compass/Compass.vue [hunk 1]
```diff
@@ -26,19 +26,20 @@
     :style="compassDimensionsStyle"
 >
     <CompassHUD
-        v-if="shouldDisplayCompassHUD"
+        v-if="hasCameraFieldOfView"
         :heading="heading"
-        :roll="roll"
         :sun-heading="sunHeading"
-        :camera-field-of-view="cameraFieldOfView"
+        :camera-angle-of-view="cameraAngleOfView"
```
Comment (akhenry) [inline src/plugins/imagery/components/Compass/Compass.vue:32]: You're right, it's not an accurate representation of field of view, and it's not configurable.

File: src/plugins/imagery/components/Compass/CompassHUD.vue [hunk 2]
```diff
@@ -99,24 +98,27 @@ export default {
             type: Number,
             required: true
         },
+        /* only useful if combined with tilt to find horizon
```
Comment (akhenry) [inline src/plugins/imagery/components/Compass/CompassHUD.vue:101]: Let's remove this commented code.

File: src/plugins/imagery/components/Compass/CompassHUD.vue [hunk 3]
```diff
@@ -99,24 +98,27 @@ export default {
...
             default: undefined
         },
         cameraPan: {
             type: Number,
-            default: undefined
+            required: true
         }
     },
     computed: {
+        /* only useful if combined with tilt to find horizon
```
Comment (akhenry) [inline src/plugins/imagery/components/Compass/CompassHUD.vue:121]: Let's remove this commented code, we have it in version control if we need it in future.

**Review by akhenry — CHANGES_REQUESTED** [review]
Looks great, just some commented code that needs removing.

File: src/plugins/imagery/components/Compass/CompassHUD.vue (hunk 2 above)
Comment (davetsay) [reply src/plugins/imagery/components/Compass/CompassHUD.vue:101]: sigh. alright.

**Comment by davetsay** [comment]
Fixed reviewer comments

@akhenry removed commented code
@jvigliotta removed options object until we use more than one option

Smoke tested

**Review by jvigliotta — APPROVED** [review]
### Reviewer Checklist
Check | Passed
-- | --
Changes appear to address issue?? | Y
Appropriate unit tests included? | N/A
Code style and in-line documentation are appropriate? | Y
Commit messages meet standards? | Y
Has associated issue been labelled unverified? (if PR closes issue) | Y

**Pull request merged**